build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -eu
+
+scriptdir=$(cd $(dirname $0) && pwd)
+
+build_cmd='cd /build &&  mvn clean package'
+
+case "$1" in
```

Comment:
a case with only one condition? If you want to do this, you should at least check for incorrect arguments.
The convention that the dev env assumes is that the default is to build, btw. Why not do that?